website/src/pages/api/admin/messages/[id]/tree/state.ts

```diff
@@ -0,0 +1,10 @@
+import { withRole } from "src/lib/auth";
+import { createApiClient } from "src/lib/oasst_client_factory";
+
+export default withRole("admin", async (req, res, token) => {
+  const client = await createApiClient(token);
+  const messageId = req.query.id as string;
+  const treeState = await client.fetch_message_tree_state(messageId);
```

Comment:
Can you move this call to the `/api/admin/messasges/[id]/tree` endpoint and remove this file?

Reply:
Sure

Reply:
Done, I committed it on the PR.